main.go

```diff
@@ -26,37 +27,59 @@ type Args struct {
 	PEMFileContents string `envconfig:"PLUGIN_PEM_FILE_CONTENTS"`
 	PEMFilePath     string `envconfig:"PLUGIN_PEM_FILE_PATH"`
 	Level           string `envconfig:"PLUGIN_LOG_LEVEL"`
+	CommitSha       string `envconfig:"DRONE_COMMIT_SHA"`
+	RepoURL         string `envconfig:"DRONE_GIT_HTTP_URL"`
+	BranchName      string `envconfig:"DRONE_REPO_BRANCH"`
+	CommitMessage   string `envconfig:"DRONE_COMMIT_MESSAGE"`
+	PLUGIN_GIT_PATH string `envconfig:"DRONE_WORKSPACE"`
```

Comment:
this is not the right way to add a input. the plugin should look for `PLUGIN_GIT_PATH` and the default value of it should be `DRONE_WORKSPACE`. @HarnessAryan could you update the code and then it's good to be merged